src/pages/Product/product.js

```diff
@@ -18,32 +18,39 @@ const useStyles = makeStyles(() => ({
 
 const Product = props => {
   const classes = useStyles();
-  const { product, fetchProduct, checkoutState, addVariant } = useShopify();
+  const {
+    product,
+    fetchProduct,
+    checkoutState,
+    addVariant,
+    openCart
```

Comment:
openCart is for the popover on the side that we will add eventually, (and it controls whether it is visible or not)
So for this page, it can be removed